src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/import_data.cljs

```diff
@@ -39,8 +39,12 @@
           (when (> (.-size (:file @state)) preview-limit)
             [:em {} "(file truncated for preview)"])]])
       (when (and (:file @state) (not (:upload-result @state)))
-        [comps/Button {:text "Upload"
-                       :onClick #(react/call :do-upload this)}])
+        [:div {:style {:display "inline-flex" }}
+         [comps/Button {:text "Done"
+                        :onClick (:cancel props)
+                        :style {:marginRight 20}}]
+         [comps/Button {:text "Upload"
+                        :onClick #(react/call :do-upload this)}]])
       (if-let [result (:upload-result @state)]
```

Comment:
Adding a "Done" button alongside the "Upload" button when importing workspace attributes